app/services/analytics_events.rb

```diff
@@ -392,9 +392,10 @@ def doc_auth_warning(message: nil, **extra)
     )
   end
 
+  # @param [Boolean] required_password_change if user forced to change password
   # When a user views the edit password page
-  def edit_password_visit
-    track_event('Edit Password Page Visited')
+  def edit_password_visit(required_password_change: false, **_extra)
```

Comment:
I'd expect the `**extra` to be logged with the tracked event, for consistency with other events.